### DISCUSSION TITLE
Is it possible to create a platform-neutral MSI?

### DISCUSSION THREAD

**Discussion by sharpjs** [wixtoolset/issues#8130] [Questions] [answered]
I've made a simple `.wxs` file for a font, which you can see [here](https://github.com/sharpjs/IosevkaSharpie/blob/main/IosevkaSharpie.wxs).  Orca's summary information view shows `Intel` as the platform of the built MSI.  The MSI installs on an x64 box, does it install on an arm64 box?  (I don't have one to test.)  If not, **is there any way via WiX to make the MSI platform-independent**, such as a `noarch` pseudo-platform?

The [relevant MSI documentation](https://learn.microsoft.com/en-us/windows/win32/msi/template-summary) suggests that the answer to both questions is 'no':

> A Windows Installer package cannot be marked as supporting both Intel64 and x64; for example, the Template Summary property value of `Intel64,x64` is invalid.
>
> A Windows Installer package cannot be marked as supporting both 32-bit and 64-bit platforms; for example, Template Summary property values such as `Intel,x64` or `Intel,Intel64` are invalid.

> The following syntax is not supported:
>
> ```
> [platform property][,platform property][,...];[language id][,language id][,...]
>                                              ↑
>                   my addition; looks like the documentation forgot to add it
> ```

**Comment by barnson** [comment, marked as answer]
MSI doesn't have the concept of an arch-neutral package. Windows supports x86 on Arm64 so an x86 package works "everywhere." Except it doesn't, because it's a Windows feature that can be turned off.

**Reply by sharpjs** [reply]
Thanks.  That tells me what I wanted to know.  Looks like one package per arch is the way to go for anyone in this situation.
